resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -14,7 +14,7 @@ if [ -r "$restartfile" ];  then
 
         ((timeout--))
         if [ "$timeout" = 0 ]; then
-            echo "Reload timed out after <%= EZbake::Config[:start_timeout] %> seconds"
+            echo "$timeout" | awk '{printf "Reload timed out after %.3f secs\n", $1/10}'
```

Comment:
@haus I was pondering changing the sleep to be a sleep variable so that I could pass it in to this instead of hardcoding the /10, but I wasn't sure if it was ever going to be changed again and thus not needed to be made into a reference. does this look ok as is to you?